control-plane/subcommand/inject-connect/command.go

```diff
@@ -196,6 +201,9 @@ func (c *Command) init() {
 		"K8s namespaces to explicitly deny. Takes precedence over allow. May be specified multiple times.")
 	c.flagSet.StringVar(&c.flagReleaseName, "release-name", "consul", "The Consul Helm installation release name, e.g 'helm install <RELEASE-NAME>'")
 	c.flagSet.StringVar(&c.flagReleaseNamespace, "release-namespace", "default", "The Consul Helm installation namespace, e.g 'helm install <RELEASE-NAME> --namespace <RELEASE-NAMESPACE>'")
+	c.flagSet.StringVar(&c.flagAppName, "app-name", "consul", "The Consul application name use when installing consul")
```

Comment:
I don't know how I feel about the descriptions for these flags, any feedback would be appreciated